operatorconfig/moduleconfig/application-mobility/v0.3.0/certificate.yaml

```diff
@@ -18,4 +21,15 @@ spec:
   issuerRef:
     kind: Issuer
     name: <NAMESPACE>-selfsigned-issuer
-  secretName: cert-manager-webhook-ca
\ No newline at end of file
+  secretName: webhook-server-cert
+status:
+conditions:
```

Comment:
We don't want status field in this template
see: https://github.com/dell/helm-charts/blob/main/charts/csm-application-mobility/templates/cert-manager.io_v1_certificate_app-mobility-serving-cert.yaml
for example

Reply:
Can you remove Conditions section and 
notAfter: "2023-10-04T15:57:27Z"
notBefore: "2023-07-06T15:57:27Z"
renewalTime: "2023-09-04T15:57:27Z"
?
That stuff should be being filled in when its created